Tweaks premium items in vending machines, allow access to contraband via emagging

PsiOmegaDelta opened the pull request:
:cl:
tweak: Replaces some premium items in vending machines, due to how prevalent coins have become.
tweak: Besides hacking, emagging now also allows access to contraband items in vending machines.
spelling: Fixes grammar issues with uppercase vending machine names.
/:cl:

SierraKomodo commented:
May want to double-check this doesn't break accessing premium good through hacking wires.

PsiOmegaDelta commented:
Wire hacking is for a separate category, contraband.

PsiOmegaDelta committed:
Adjust vending machines

Replaces some too-high grade premium items
Emagging now also grants access to contraband items
Fixes grammar issues with uppercased names

PsiOmegaDelta commented:
Adjustments to original PR made.
